zstd-safe/src/lib.rs

```diff
@@ -804,6 +804,45 @@ impl Default for DCtx<'_> {
     }
 }
 
+#[cfg(feature = "experimental")]
+pub fn read_skippable_frame<C: WriteBuf + ?Sized>(dst: &mut C, magic_variant: &mut u32, input: &[u8]) -> SafeResult {
+    let input_len = input.len();
+    unsafe {
```

Comment:
I suggest adding SAFETY comments for the unsafe blocks and wrapping the minimum amount of code in the `unsafe` block. See https://rust-lang.github.io/rust-clippy/master/#/undocumented_unsafe_blocks